### PR TITLE
Play Json generator

### PR DESCRIPTION
**Pull request by benwaffle**

### CODE REVIEW

File: src/main/scala/rocks/muki/graphql/codegen/ApolloSourceGenerator.scala [hunk 1]
```diff
@@ -134,7 +133,6 @@ case class ApolloSourceGenerator(
            ..$data
           }"""
     }
-    val types = document.types.flatMap(generateType)
```
Comment (benwaffle) [inline src/main/scala/rocks/muki/graphql/codegen/ApolloSourceGenerator.scala:137]: unused

File: src/main/scala/rocks/muki/graphql/codegen/ApolloSourceGenerator.scala [hunk 2]
```diff
@@ -210,12 +208,12 @@ case class ApolloSourceGenerator(
             val innerSelections = unionSelection.fields.flatMap { field =>
               selectionStats(field, List.empty)
             }
+            val unionTypeName = Type.Name(unionType.name)
+            val unionTermName = Term.Name(unionType.name)
             val params = generateFieldParams(
               unionSelection.fields,
-              typeQualifiers :+ unionName.value
+              typeQualifiers :+ unionTermName.value
```
Comment (benwaffle) [inline src/main/scala/rocks/muki/graphql/codegen/ApolloSourceGenerator.scala:215]: This fixes a bug where the case class field type is incorrect

Comment (benwaffle) [reply src/main/scala/rocks/muki/graphql/codegen/ApolloSourceGenerator.scala:215]: TODO: add a test for this

**Comment by muuki88** [comment]
Wow :astonished: :heart_eyes:  Thanks a lot for this amazing contribution and thanks for your patience. Once this PR is green I'll merge and release

File: src/main/scala/rocks/muki/graphql/codegen/ApolloSourceGenerator.scala (hunk 2 above)
Comment (muuki88) [reply src/main/scala/rocks/muki/graphql/codegen/ApolloSourceGenerator.scala:215]: I'm happy if you add a test case, but this is no blocker. So if you can add one nice, if you say you don't have the time, then I'll merge and hope for second PR at some point in time :grin:

**Comment by muuki88** [comment]
`0.17.0` release is on its way

**Pull request merged**